tests/conftest.py

```diff
@@ -117,6 +118,20 @@ class LiveLoggingStreamHandler(
 
 
 # ----- PyTest Tempdir Plugin Hooks --------------------------------------------------------------------------------->
+def pytest_tempdir_temproot():
```

Comment:
This change is no longer required 

Latest version of salt-factories has this issue addressed.

https://github.com/saltstack/pytest-salt-factories/blob/0.906.0/src/saltfactories/plugins/__init__.py#L19

Reply:
So, I can just delete this then?

Reply:
Yes, please